wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
@mikekaganski So, yes, for the specific function in use, a local buffer would also work fine as the function is not really intended to be run tenth of thousands of times.
But as there is already the global buffer present, I found that replacing only the needed part in the buffer was a more elegant solution (once this `0x00` has been taken care of).